[2023-03-13 16:10:30,976][DEBUG]: Starting new HTTP connection (1): localhost:8080
[2023-03-13 16:10:30,982][DEBUG]: http://localhost:8080 "GET / HTTP/1.1" 404 None
[2023-03-13 16:10:30,984][DEBUG]: Reproduce HTTP request with:
[2023-03-13 16:10:30,984][DEBUG]: curl -X POST 'http://localhost:8080/evaluateFromRepoWithParams' --data 'programName=../../../../disk/results/chasebench/ontology-256/vadalog-parsimonious-aggregate/data/q01/run-1/new_program.vada' --data 'params=NonExist=1' --data 'prop=NonExistingProp=0'
[2023-03-13 16:10:30,985][DEBUG]: Starting new HTTP connection (1): localhost:8080
[2023-03-13 16:10:40,164][DEBUG]: http://localhost:8080 "POST /evaluateFromRepoWithParamsProp HTTP/1.1" 200 None
